tests/test_sunsch.py

```diff
@@ -181,23 +181,6 @@ def test_templating(tmp_path):
     assert "200.3" in str(sch)
     assert "1400000" in str(sch)
 
-    # Let some of the valued be undefined:
-    sunschconf = {
-        "startdate": datetime.date(2020, 1, 1),
-        "enddate": datetime.date(2021, 1, 1),
-        "insert": [
-            {
-                "template": "template.tmpl",
-                "days": 10,
-                "substitute": {"WELLNAME": "A-007"},
-            }
-        ],
-    }
-    sch = sunsch.process_sch_config(sunschconf)
-    assert "A-007" in str(sch)
-    assert "<ORAT>" in str(sch)
-    # (this error is let through sunsch)
-
     # Let the date be undefined.
```

Comment:
`opm` doesn't accept <ORAT> argument anymore. Previously, it was taken as UDQ, but now it's flagged as invalid UDQ (with the new name validation)